.github/workflows/staging_pipeline.yml

```diff
@@ -80,9 +80,9 @@ jobs:
           # runs 'bundle install' and caches installed gems automatically
           bundler-cache: true
 
-      - name: Rubocop
+      - name: Lint
         run: |
-          bundle exec rubocop
+          bundle exec rake lint
```

Comment:
Do we want a single linting task in this pipeline, or separate steps for each linter?

Reply:
Single task is fine I think